SECURITY.md

```diff
@@ -0,0 +1,8 @@
+# Security Policy
+
+### Reporting a vulnerability
+
+Please do not open GitHub issues or pull requests - this makes the problem immediately visible to everyone, including malicious actors.   
```

Comment:
Maybe say "Please do not open GitHub issues or pull requests when there is a vulnerability" just so it's explicit we mean in that scenario. We do want people to add in GH issues for general requests.

Reply:
Discussed out-of-band. This is standard verbiage from Stripe and changes should go through the central process, not be one-offs.

Reply:
Updated.